src/ironoak/person/PersonDetector.py

```diff
@@ -0,0 +1,168 @@
+
```

Comment:
Once you make the code work for live feed of the camera, you could take a look at the following formatting comments.

I would prefer not to use camel case for filenames. filenames can be person_detector.py. This is unix style.

For other formatting guidance you try using https://google.github.io/styleguide/pyguide.html
To automatically format your file, 

you can try `black` formatter.
`pip install git+https://github.com/psf/black`

Reply:
I usually run black -l 120 <filename>.py for 120 characters per line.